introduction.md

```diff
@@ -0,0 +1 @@
+Introduciton MD
```

Comment:
If you copy and paste the codeschool-kiddo text into your file update then you get a successful task completion.